src/irr/asset/CGLSLLimitsBuiltinIncludeLoader.h

```diff
@@ -39,7 +39,7 @@ R"(#ifndef _IRR_LIMITS_NUMERIC_INCLUDED_
     core::vector<std::pair<std::regex, HandleFunc_t>> getBuiltinNamesToFunctionMapping() const override
     {
         return {
-            { std::regex{"numeric\\.glsl"}, &getNumericLimits }
+            { std::regex{"glsl/limits/numeric\\.glsl"}, &getNumericLimits }
```

Comment:
you actually don't need there classes anymore, just plop your shaders in the correct builtin path, and add to builtin CMakeLists.txt